src/components/pages/SideDrawer/SideDrawer.js

```diff
@@ -125,57 +96,30 @@ export default function SideDrawer(props) {
     };
   }, [mobile]);
 
-  const handleDrawerOpen = () => {
-    setOpen(true);
-  };
-
-  const handleDrawerClose = () => {
-    setOpen(false);
+  const handleDrawer = () => {
+    setOpen(!open);
   };
 
   return (
     <div className={classes.root}>
       <CssBaseline />
-      <AppBar
-        elevation={0}
-        position="fixed"
-        className={clsx(classes.appBar, {
-          [classes.appBarShift]: open,
-        })}
-      >
-        <Toolbar>
-          <IconButton
-            color="inherit"
-            aria-label="open drawer"
-            onClick={handleDrawerOpen}
-            edge="start"
-            className={clsx(classes.menuButton, open && classes.hide)}
-          >
-            <MenuIcon />
-          </IconButton>
-          <Typography>
-            <a href="/">
-              <img src={HRFlogo} alt=" HRF logo" className="hrflogo" />
-            </a>
-          </Typography>
-        </Toolbar>
-      </AppBar>
       <Drawer
-        className={classes.drawer}
-        variant="persistent"
-        anchor="left"
-        open={open}
+        variant="permanent"
+        className={clsx(classes.drawer, {
+          [classes.drawerOpen]: open,
+          [classes.drawerClose]: !open,
+        })}
         classes={{
-          paper: classes.drawerPaper,
+          paper: clsx(classes.drawerPaper, {
+            [classes.drawerOpen]: open,
+            [classes.drawerClose]: !open,
+          }),
         }}
       >
-        <div className={classes.drawerHeader}>
-          <IconButton onClick={handleDrawerClose}>
-            {theme.direction === 'ltr' ? (
-              <ChevronLeftIcon />
-            ) : (
-              <ChevronRightIcon />
-            )}
+        <div className={classes.toolbar}>
+          <IconButton onClick={handleDrawer}>
+            {open === false ? <ChevronRightIcon /> : <ChevronLeftIcon />}
+            {/* {open === false ? <ArrowRight /> : <ArrowLeft />} */}
```

Comment:
Did you want to leave this commented code in?

Reply:
Yes! I wanted to get back to this tomorrow when I try importing the icons. Thanks for asking!

Reply:
Perfect!